### DISCUSSION TITLE
Layout Drawer top -> bottom collapse

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [quasarframework/quasar#9069] [General - Components / Directives / etc]
It´s possible to do a drawer menu top down like the video? Now, i can make this menu drawer right left or left right. But top down, i don´t know if it´s possible with Quasar. Any tip to do this menu type?

https://user-images.githubusercontent.com/379850/116314662-c3a82e00-a785-11eb-9c8e-d7aeb3b1e918.mov

**Comment by hawkeye64** [comment]
@hermesalvesbr  Use QExpansionItem https://next.quasar.dev/vue-components/expansion-item
